lib/src/create_report.dart

```diff
@@ -264,9 +230,10 @@ Future<ReportSection> _followsTemplate(
   }
 
   // TODO(sigurdm): What do we want to check here?
```

Comment:
I thought we had some checks for pubspec.yaml already (things like max lengths for certain fields, Urls has proper format)?